sass/_twitter-night-mode.scss

```diff
@@ -494,6 +494,10 @@ body.logged-out {
 	background-color: $dark-bg!important;
 }
 
+.AppContent:not(.wrapper){
```

Comment:
Why did you do it like this and not like this: 

.MomentsPage.MomentsPermalinkPage, .MomentsPage.MomentsGuidePage {
    background-color: #2c4158 !important;
}